types/google-cloud__storage/index.d.ts

```diff
@@ -17,7 +18,7 @@ declare namespace Storage {
     class Bucket {
         constructor(storage: Storage, name: string);
         acl: Acl;
-        combine(sources: string[] | File[], destination: string[] | File[]): Promise<[File, ApiResponse]>;
```

Comment:
you have a syntax error.. add back the closing `)`

Reply:
Fixed